guides/v2.1/coding-standards/layout-coding-standard.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: default
+group: coding-standards
+subgroup: Coding standards
+title: Layout XML Coding Standards
+menu_title: Layout XML Coding Standards
+menu_order: 100
+contributor_name: SwiftOtter Studios
+contributor_link: https://swiftotter.com/
+version: 2.1
+github_link: coding-standards/layout-coding-standard.md
+---
+
+# Layout XML Coding Standard
+
+### Name attribute
+
+The `name` attribute should adhere to the guidelines listed below. The `name` attribute should:
+
+- Should explain element's purpose rather than its position on the page .
+- Be unique to the module that declares it by prepending it with module's namespace as `<vendor>.<module>`
```

Comment:
I think I like this idea. However, I'm a bit hesitant because literally none of the core does this. I was hoping to keep it compatible with at least the general principle of what is already in the core.

Reply:
It can be done for the new layouts. So far it's just my idea, I'll be discussing it more. Thanks for your feedback.